<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="streamTV.TestNG by packages" duration-ms="268118" started-at="2016-04-13T07:16:49Z" finished-at="2016-04-13T07:21:17Z">
    <groups>
    </groups>
    <test name="streamTV" duration-ms="268118" started-at="2016-04-13T07:16:49Z" finished-at="2016-04-13T07:21:17Z">
      <class name="lastochkin.streamTV.RegisterTest">
        <test-method status="PASS" signature="Filters()[pri:0, instance:lastochkin.streamTV.RegisterTest@313b2ea6]" name="Filters" duration-ms="101982" started-at="2016-04-13T10:16:49Z" finished-at="2016-04-13T10:18:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Filters -->
        <test-method status="PASS" signature="createAndDelete()[pri:0, instance:lastochkin.streamTV.RegisterTest@313b2ea6]" name="createAndDelete" duration-ms="44027" started-at="2016-04-13T10:18:31Z" finished-at="2016-04-13T10:19:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createAndDelete -->
        <test-method status="FAIL" signature="createAndVerify()[pri:0, instance:lastochkin.streamTV.RegisterTest@313b2ea6]" name="createAndVerify" duration-ms="44203" started-at="2016-04-13T10:19:15Z" finished-at="2016-04-13T10:20:00Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Some fields contain wrong data!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Some fields contain wrong data!
	at lastochkin.streamTV.pages.HomePage.checkExeptions(HomePage.java:124)
	at lastochkin.streamTV.RegisterTest.createAndVerify(RegisterTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1189)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createAndVerify -->
        <test-method status="FAIL" signature="updateAndVerify()[pri:0, instance:lastochkin.streamTV.RegisterTest@313b2ea6]" name="updateAndVerify" duration-ms="77645" started-at="2016-04-13T10:20:00Z" finished-at="2016-04-13T10:21:17Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Some fields contain wrong data!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Some fields contain wrong data!
	at lastochkin.streamTV.pages.HomePage.checkExeptions(HomePage.java:124)
	at lastochkin.streamTV.RegisterTest.updateAndVerify(RegisterTest.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1189)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateAndVerify -->
      </class> <!-- lastochkin.streamTV.RegisterTest -->
    </test> <!-- streamTV -->
  </suite> <!-- streamTV.TestNG by packages -->
</testng-results>
